deploy/clowdapp.yaml

```diff
@@ -268,6 +268,27 @@ objects:
           requests:
             cpu: 100m
             memory: 64Mi
+    - name: org-id-populator
+      podSpec:
+        name: org-id-populator
```

Comment:
This does not seem entirely correct according to the [Clowdapp schema definition](https://consoledot.pages.redhat.com/clowder/dev/api_reference.html#k8s-api-github-com-redhatinsights-clowder-apis-cloud-redhat-com-v1alpha1-job) (e.g. podspec does not have a name, restart policy should be job-level, not podspec-level attribute)